### PR TITLE
polish docs

### PR DESCRIPTION
**Pull request by dmsolr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
-Apache SkyWalking Nginx Agent For Kong
+Apache SkyWalking Kong Agent
 ==========
 
-This plugin base on Apache SkyWalking Nginx Agent for the Kong API gateway to integrate with the Apache SkyWalking distributed tracing system.
+<img src="http://skywalking.apache.org/assets/logo.svg" alt="Sky Walking logo" height="90px" align="right" />
+
+[![Twitter Follow](https://img.shields.io/twitter/follow/asfskywalking.svg?style=for-the-badge&label=Follow&logo=twitter)](https://twitter.com/AsfSkyWalking)
+
+![CI](https://github.com/apache/skywalking-nginx-lua/workflows/CI/badge.svg?branch=master)
```
Comment (kezhenxu94) [inline README.md]: Wrong link

File: README.md [hunk 2]
```diff
@@ -48,4 +55,13 @@ $ curl -X POST --url http://localhost:8001/plugins/ \
    --data 'config.sample_ratio=100' \
    --data 'config.service_name=kong'
    --data 'config.service_instance_name=kong-with-skywalking'
-```
\ No newline at end of file
+```
+
+# Contact Us
+* Submit an [issue](https://github.com/apache/skywalking/issues) with `[NIGNX-LUA]` as the issue title prefix.
```
Comment (kezhenxu94) [inline README.md]: Wrong prefix

**Comment by wu-sheng** [comment]
Please add .asf control file.

File: README.md (hunk 2 above)
Comment (dmsolr) [reply README.md]: what do you think about `KONG_LUA` as prefix?

**Comment by dmsolr** [comment]
> Please add .asf control file.

added.

File: README.md [hunk 3]
```diff
@@ -58,10 +58,10 @@ $ curl -X POST --url http://localhost:8001/plugins/ \
 ```
 
 # Contact Us
-* Submit an [issue](https://github.com/apache/skywalking/issues) with `[NIGNX-LUA]` as the issue title prefix.
+* Submit an [issue](https://github.com/apache/skywalking/issues) with `[KONG_LUA]` as the issue title prefix.
```
Comment (wu-sheng) [inline README.md]:
```suggestion
* Submit an [issue](https://github.com/apache/skywalking/issues) with `[KONG]` as the issue title prefix.
```

**Comment by wu-sheng** [comment]
> > Please add .asf control file.
> 
> added.

We need CI passed required and review required(1).

**Pull request merged**